docs/content_management/images/fastly_io.md

```diff
@@ -0,0 +1,147 @@
+---
+description: Configure Fastly Image Optimiser.
+---
+
+# Fastly Image Optimiser (Fastly IO)
+
+The Fastly Image Optimizer (Fastly IO) is an external service that provides real-time image optimisation for multiple input and output formats.
+It serves and caches image requests from your origin server, making your website faster and more efficient.
+
+To be able to configure this feature you need:
+
+- [Platform.sh project](https://docs.platform.sh/guides/ibexa/deploy.html)
+- [Fastly IO subscription](https://docs.fastly.com/en/guides/about-fastly-image-optimizer)
+
+## Enable shielding
+
+To use Fastly Image Optimizer, you need shielding, for more details
+see [Enabling and disabling shielding](https://developer.fastly.com/learning/concepts/shielding/).
+
+To enable shielding, add [`snippet_re_enable_shielding.vcl`](https://github.com/ibexa/fastly/blob/main/fastly/snippet_re_enable_shielding.vcl) snippet to your config.
+
+Next, use command below in Fastly CLI: 
+ 
+```bash
+- fastly vcl snippet create --name="Re-Enable shielding on restart" --version=active --autoclone --priority 100 --type recv --content=vendor/ibexa/fastly/fastly/snippet_re_enable_shielding.vcl
+- fastly service-version activate --version=latest
+```
+
+## VCL configuration
+
+To manipulate your Fastly VCL configuration directly from the command line,
```

Comment:
I think this paragraph should somehow mention the new image optimizer vcl:
https://github.com/ibexa/fastly/blob/main/fastly/ibexa_image_optimizer.vcl